aviary/docs/user_guide/troubleshooting.ipynb

```diff
@@ -74,11 +110,40 @@
     "\n",
     "This section is under development.\n"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Testing Cell\n",
+    "# ********NOTE********\n",
+    "# Should we check that traj_results_report.html and opt_report.html are the right names?"
```

Comment:
Probably not strictly necessary; I don't think those names will change any time soon